model/QueueBroker/RdsQueueBroker.php

```diff
@@ -233,6 +235,49 @@ protected function doDelete($id, array $logContext = [])
         }
     }
 
+    public function getTaskByTaskLogId(string $taskLogId): ?CallbackTaskDecorator
```

Comment:
What about other classes inhered from `AbstractQueueBroker`?  If we can't go with a proper implementation may be at least something like `throw new NotSupportedMenthodExpeption()`  to maintain integrity ?

Reply:
The other implementations like SQS do not work with ids. This implementation is really to support RDS. I will open a tech debt task for this after we validate the solution.

Reply:
Tech debt here: https://oat-sa.atlassian.net/browse/ADF-556